ClangSharpPInvokeGenerator/Program.cs

```diff
@@ -12,7 +12,7 @@ public class Program
     {
         private static RootCommand s_rootCommand;
 
-        public static async Task<int> Main(string[] args)
+        public static async Task<int> Main(params string[] args)
```

Comment:
It might be nice to make the `PInvokeGenerator` a simple wrapper over some library APIs. That would make it easier to test and for other people to use/extend...